rad/src/script.rs

```diff
@@ -211,10 +213,17 @@ fn errorify(kind: RadError) -> RadError {
 pub fn create_radon_script_from_filters_and_reducer(
     filters: &[RADFilter],
     reducer: u32,
+    active_wips: &Option<ActiveWips>,
```

Comment:
Clippy should detect this: prefer using `Option<&ActiveWips>` here, and in the callsite use `.as_ref()` to perform the conversion.